client/my-sites/checkout/checkout/composite-checkout-wpcom-container.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { createRegistry, createPayPalMethod } from '@automattic/composite-checkout';
+import {
+	WPCOMCheckout,
+	makeShoppingCartHook,
+	callWpcomShoppingCartEndpoint,
+	mockPayPalExpressRequest,
+} from '@automattic/composite-checkout-wpcom';
+
+/**
+ * Internal dependencies
+ */
+import wp from 'lib/wp';
+
+const initialCart = {
+	coupon: '',
+	currency: 'BRL',
+	is_coupon_applied: false,
+	products: [
+		{
+			extra: {
+				context: 'signup',
+				domain_registration_agreement_url:
+					'https://wordpress.com/automattic-domain-name-registration-agreement/',
+				privacy: true,
+				privacy_available: true,
+				registrar: 'KS_RAM',
+			},
+			free_trial: false,
+			meta: 'foo.cash',
+			product_id: 106,
+			volume: 1,
+		},
+		{
+			extra: {
+				context: 'signup',
+				domain_to_bundle: 'foo.cash',
+			},
+			free_trial: false,
+			meta: '',
+			product_id: 1009,
+			volume: 1,
+		},
+	],
+	tax: {
+		display_taxes: false,
+		location: {},
+	},
+	temporary: false,
+};
+
+const registry = createRegistry();
+const { registerStore } = registry;
+
+const wpcom = wp.undocumented();
+
+const useShoppingCart = makeShoppingCartHook( callWpcomShoppingCartEndpoint( wpcom ), initialCart );
+
+export default function CompositeCheckoutWPCOMContainer() {
```

Comment:
Let's just move this all over to `composite-checkout-container.js`, replace what's there, and rename this component `CompositeCheckoutContainer`.